cmd/ak/cmd/manifest/apply.go

```diff
@@ -33,9 +33,11 @@ var applyCmd = common.StandardCommand(&cobra.Command{
 			ctx, cancel := common.LimitedContext()
 			defer cancel()
 
-			return imanifest.Execute(ctx, actions, common.Client(), func(msg string) {
+			pids, err := imanifest.Execute(ctx, actions, common.Client(), func(msg string) {
 				fmt.Fprintf(os.Stderr, "[exec] %s\n", msg)
 			})
+			fmt.Println(pids)
```

Comment:
I'm OK with adding the data to the API for the extension, but do you think it's important to print it in the CLI too?

1. I'm OK with `os.Fprintf` for logs, but data printing is done in all commands with `common.Render` or a variant for lists
2. Combining stderr and stdout is usually a bad idea
3. It's unclear to the user what this line actually means, and it's not contributing new information on top of the exec logs